{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/consumers.py

```diff
@@ -0,0 +1,27 @@
+from channels.generic.websocket import AsyncJsonWebsocketConsumer
+
+
+class PushNotificationConsumer(AsyncJsonWebsocketConsumer):
```

Comment:
can you give docstrings for the methods on this class? how the parameters are actually used seems nonintuitive

Reply:
All of this comes from things in Channels; if you've been living in the docs for Channels, it seems fine. I say that only to say: do we want to assume people using this may _not_ be living in those docs, and give orientation? I think that's reasonable—this is just to gauge intended audience.